### PR TITLE
Comparaison nanoduration and character

### PR DESCRIPTION
**Pull request by statquant**
Following [#87](https://github.com/eddelbuettel/nanotime/issues/87) here is a PR that allows comparaison between nanoduration and character. Functionality-wise I think it makes sense as it mimics nanotime behavior and base R Date behavior.
It builds locally and passes all tinytests.
I had to modify 2 tests that would have failed.
If you're ok with adding the functionality please let me know if I should check/change/fix anything.
Regards

### CODE REVIEW

**Comment by eddelbuettel** [comment]
Thanks!  From a first glance that looks spot on.  (I even saw you fork and looked briefly and saw the first commit so thanks for the second -- my use of `roxygen2` would just have overwritten ...)

As you know I tend to document each change in each file (apart from very trivial ones) via ChangeLog.  If you would like me to use an email that different than the one that comes up (with your work address) let me know.   An anonymous PR will not go as that is simply not how we roll. Users of the code deserve to know where it comes from.

**Comment by eddelbuettel** [comment]
(Oh, btw,  _la comparaison_ becomes the _the comparison_.  I fight with English too as a non-native speaker...)

File: inst/tinytest/test_nanoduration.R [hunk 1]
```diff
@@ -385,7 +385,7 @@ expect_true(is.na(as.nanoduration(1) < "a")) # this is what "integer64" gives ba
 ## [1] TRUE  
 
 ##test_Compare_ANY_nanoduration <- function() {
-expect_true(is.na("a" < as.nanoduration(1))) # this is what "integer64" gives back, but do we want to change that to an error? LLL
+expect_error("a" < as.nanoduration(1), "cannot parse nanoduration")
```
Comment (eddelbuettel) [inline inst/tinytest/test_nanoduration.R:388]: Could please add two tests actually testing the new functionality?

**Review by eddelbuettel — CHANGES_REQUESTED** [review]

**Comment by statquant** [comment]
Hello,
With regard to required tests: 
I have added 2 sets of tests testing for `==`, `!=`, `>`, `>=`, `<`, `<=` and they passed locally
With regard to ChangeLog:
Thanks for bringing the email to my attention, I rebased and changed the email to the one being displayed on my profile. My understanding is that you require my name in `git log`, so I left it.
Thanks

**Review by eddelbuettel — APPROVED** [review]

**Comment by eddelbuettel** [comment]
Looks good, thanks!  I had the pr checked out here (emacs magit makes that so easy...), and your rebase/force broke that but I just started over.

Thanks for adding those tests, that is very nice too.  I'll look into adding a quick changelog entry.  And will look into the failed Travis builds, looks like a spurious setup issue (which happens once in a while).  It passes locally for me here.

**Comment by eddelbuettel** [comment]
I think we're there---I keep pushing into your branch now for simplicity.  My Travis CI credits may be up, so I am now adding a runner at GitHub Actions.

**Comment by eddelbuettel** [comment]
Ok, and rolled the minor version to distinguish the tar ball.

Your corresponding ChangeLog change is in there too.

**Pull request merged**